apps/dashboard/.env.docker

```diff
@@ -0,0 +1,10 @@
+NEXT_PUBLIC_STACK_URL=http://localhost:8102
+
+NEXT_PUBLIC_STACK_PROJECT_ID=internal
+NEXT_PUBLIC_STACK_PUBLISHABLE_CLIENT_KEY=this-publishable-client-key-is-for-local-development-only
+STACK_SECRET_SERVER_KEY=this-secret-server-key-is-for-local-development-only
+STACK_SUPER_SECRET_ADMIN_KEY=this-super-secret-admin-key-is-for-local-development-only
```

Comment:
Those are bad default values. Do we need them? Can we just throw an error when people try to build a Docker image without explicitly specifying all environment variables, and then get rid of the `.env.docker` files?

Reply:
This is also a file that I forgot to delete